modules/fbx/extensions/SCsub

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+
+Import("env")
+Import("env_modules")
+
+env_gltf = env_modules.Clone()
+
+# Godot source files
+
+env_gltf.add_source_files(env.modules_sources, "*.cpp")
+#env_gltf.add_source_files(env.modules_sources, "physics/*.cpp")
```

Comment:
```suggestion
#env_gltf.add_source_files(env.modules_sources, "physics/*.cpp")
```
```suggestion
env_gltf.add_source_files(env.modules_sources, "*.cpp")
```

Reply:
this broke the building process reverting changes